### PR TITLE
[ELY-2520] Fix system property resolution in JSON files

### PR DESCRIPTION
**Pull request by fjuma**
https://issues.redhat.com/browse/ELY-2520

### CODE REVIEW

**Review by skyllarr — APPROVED** [review]

File: jose/util/src/main/java/org/wildfly/security/jose/util/SystemPropertiesJsonParserFactory.java [hunk 1]
```diff
@@ -97,7 +97,7 @@ public String getProperty(String key) {
                 return overrides.get(key);
             } else if (key.startsWith("env.")) {
                 return System.getenv().get(key.substring(4));
-            } else if (System.getProperties().contains(key)) {
+            } else if (System.getProperties().containsKey(key)) {
```
Comment (darranl) [inline jose/util/src/main/java/org/wildfly/security/jose/util/SystemPropertiesJsonParserFactory.java:100]: No a reason to hold this PR but when is this called?  Just during deployment or from an app as well?

Just wondering if there is a security manager implication.

**Review by darranl — APPROVED** [review]

File: jose/util/src/main/java/org/wildfly/security/jose/util/SystemPropertiesJsonParserFactory.java (hunk 1 above)
Comment (fjuma) [reply jose/util/src/main/java/org/wildfly/security/jose/util/SystemPropertiesJsonParserFactory.java:100]: This is called during servlet context initialization.

Comment (darranl) [reply jose/util/src/main/java/org/wildfly/security/jose/util/SystemPropertiesJsonParserFactory.java:100]: Would that mean the deployments protection domain is on the call stack at that time?

Comment (fjuma) [reply jose/util/src/main/java/org/wildfly/security/jose/util/SystemPropertiesJsonParserFactory.java:100]: Just checked, this gets called just during deployment - the deployment's protection domain is not on the call stack at the time this is called.

**Pull request merged**